tests/drivers/i2c/i2c_slave_eeprom/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+macro(set_conf_file)
+  if(EXISTS               ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf)
+    set(CONF_FILE "prj.conf ${APPLICATION_SOURCE_DIR}/boards/${BOARD}.conf")
+  else()
+    set(CONF_FILE "prj.conf")
+  endif()
```

Comment:
There exists a convention that board-specific .conf files are named

 ${APPLICATION_SOURCE_DIR}/prj_${BOARD}.conf

The build system is aware of this convention and will use it automatically to detect board-specific .conf files.

So if you

mv boards/nucleo_f091rc.conf prj_nucleo_f091rc.conf

then you can get rid of this macro and it will just-work.

Reply:
It's the same issue I want to solve as in #5200, where I want a commpon .conf and a board specific .conf fragment as @mbolivar suggested.